meta-mender-core/recipes-mender/mender-artifact/mender-artifact.bb

```diff
@@ -7,16 +7,16 @@ inherit allarch
 PV = "0.1"
 
 do_compile() {
-    echo "# populate this file with build info" > ${B}/artifact-info
+    cat > ${B}/artifact-info << END
+artifact_name=${MENDER_ARTIFACT_NAME}
```

Comment:
I am not sure if this is a better way than providing the file and copy this one instead of creating this on the fly. This might be a bit harder to extend the file with extra information. But this is OK for now I suppose.